src/main/java/io/github/glasspane/basemod/BaseMod.java

```diff
@@ -1,4 +1,4 @@
-package com.github.glasspane.basemod;
+package io.github.glasspane.basemod;
```

Comment:
why the chane to `io.` instead of `com.`?
plus it's not consistent over all files,